src/content/docs/references/v2/acl.mdx

```diff
@@ -0,0 +1,197 @@
+  ]
+}
+```
+
+### Target Platform
+
+Capabilities can be platform-specific by defining the `platforms` array.
+By default the capability is applied to all targets,
+but you can select a subset of the `linux`, `macOS`, `windows`, `iOS` and `android` targets.
+
+For example, let's define a capability for desktop. Note it enables permissions on plugins that are only available on desktop:
+
+```json title="src-tauri/capabilities/desktop.json
+{
+  "$schema": "./schemas/desktop-schema.json",
+  "identifier": "desktop-capability",
+  "windows": ["main"],
+  "platforms": [
+    "linux",
+    "macOS",
+    "windows"
+  ],
+  "permissions": [
+    "global-shortcut:allow-register"
+  ]
+}
+```
+
+And define a capability for mobile. Note it enables permissions on plugins that are only available on desktop:
+
+```json title="src-tauri/capabilities/mobile.json
+{
+  "$schema": "./schemas/mobile-schema.json",
+  "identifier": "mobile-capability",
+  "windows": ["main"],
+  "platforms": [
+    "iOS",
+    "android"
+  ],
+  "permissions": [
+    "nfc:allow-scan",
+    "biometric:allow-authenticate",
+    "barcode-scanner:allow-scan"
+  ]
+}
+```
+
+### JSON Schema
+
+Tauri generates a JSON schema with all the permissions that are available to your application so you have autocompletion on your IDE.
+To use the schema, set the `$schema` property to one of the schemas inside the `capabilities/schemas` directory, which are platform-specific.
+Usually you will set it to `./schemas/desktop-schema.json` or `./schemas/mobile-schema.json` though you can also define a capability for a specific target platform.
+
+In the example above we defined specific capabilities for desktop and mobile, and referenced the associated JSON schema. Let's define a capability only for Linux:
+
+```json title="src-tauri/capabilities/linux.json
+{
+  "$schema": "./schemas/linux-schema.json",
+  "identifier": "linux-capability",
+  "windows": ["main"],
+  "platforms": [
+    "iOS",
+    "android"
+  ],
+  "permissions": [
+    "dbus::call"
+  ]
+}
+```
+
+### Multiwindow
+
+The `windows` property defines the list of windows that should reference the list of permissions. You can use glob patterns to reference dynamic window labels.
+
+The following example links the capability with the `main` window and any window with label prefixed with `editor-`.
+
+```json title="src-tauri/capabilities/mobile.json
+{
+  "windows": ["main", "editor-*"],
+  "permissions": [
+    ...
+  ]
+}
+```
+
+### Scopes
+
+Scopes further restrict the commands. Their format are plugin-specific, but Tauri offers primitives to allow and deny certain actions of the command, usually restricting command input.
+For instance, the `fs` plugin allows you to use scopes to allow or deny certain directories and files, and the `http` plugin uses scopes to filter URLs that are allowed to be reached.
+
+Let's study some examples for official plugins:
+
+- allow any `fs` command to access the app directory:
```

Comment:
The $APP variable was deprecated in 1.x - or did we add a new one? Didn't keep track of it tbh.

On that note, maybe it makes sense to have it point to the install dir? That's what many assumed it does by the name. Or maybe we should still not add this since the only bundle type where this is supported (nsis perUser) is covered by `$APPLOCALDATA`.